src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/EventServiceTest.java

```diff
@@ -39,7 +39,7 @@
 import uk.gov.ons.ctp.response.collection.exercise.service.EventService.Tag;
 import uk.gov.ons.response.survey.representation.SurveyDTO;
 
-/** Class containing tests for EventServiceImpl */
+// ** Class containing tests for EventServiceImpl */
```

Comment:
Should be `/** Class containing tests for EventServiceImpl **/` No need for the `//`